File: go/course-repo/exercise/05-heartbeats-and-checkpointing/webhook/shared.go
package webhook

// TaskQueue is the queue the Worker polls and the client submits to.
const TaskQueue = "webhook-queue"

// WebhookReceiverURL is the local receiver that records deliveries.
const WebhookReceiverURL = "http://localhost:9000/hooks"

// WebhookDeliveryBatch is the input to DeliverWebhookBatch: a list of items to
// POST one at a time, with per-item progress checkpointed via Heartbeat.
type WebhookDeliveryBatch struct {
	URL   string           `json:"url"`
	Items []map[string]any `json:"items"`
}

File: java/course-repo/exercise/02-idempotency-and-crash-safety/src/main/java/webhook/Shared.java
package webhook;

/** Constants shared by the Worker, the client starters, and the Activity. */
public final class Shared {
    /** The Task Queue the Worker polls and the client submits to. */
    public static final String TASK_QUEUE = "webhook-queue";

    /** The local receiver that records deliveries. */
    public static final String WEBHOOK_RECEIVER_URL = "http://localhost:9000/hooks";

    private Shared() {}
}
